lib/app/services/gruene_api_action_area_service.dart

```diff
@@ -20,4 +21,11 @@ class GrueneApiActionAreaService extends GrueneApiBaseService {
       ),
     ),
   );
+
+  Future<void> updateActionAreaAssignemnt(ActionAreaAssignmentUpdateModel actionArea) => getFromApi(
+    apiRequest: (api) => api.v1CampaignsAreasAreaIdTeamPut(
+      areaId: actionArea.id,
+      body: AreaAssignTeam(teamId: actionArea.team?.id),
```

Comment:
Maybe this is a stupid question, but what happens if the `team` prop is falsy?

Reply:
Not stupid. It's called a "null-shorting cascade" means, when a thing is null (which is marked that way) it returns null. `team` in that case is nullable, and we can reset the team when needed. For transport we just use the `teamId` and with the `?.` we get this null when we need it and save some extra ops a equivalent thing would be: `actionArea.team == null ? null ? actionArea.team.id`